alpha/notify-worker/app/process-notifications.js

```diff
@@ -0,0 +1,34 @@
+var logger = require('./bunyan-logger')
+var client = require('./db-client')
+var mailer = require('./mailer')
+var Promise = require('bluebird')
+
+const EMAIL_CONTACT_METHOD = 'email'
+
+exports.run = function () {
+  client.getPendingNotifications()
+    .then(function (notifications) {
+      if (notifications) {
+        Promise.all(notifications.map(function (notification) {
+          processNotification(notification)
+        }))
+      }
+    })
+    .catch(function (error) {
+      logger.error(error)
+    })
+}
+
+var processNotification = function (notification) {
```

Comment:
Should we separated out, with process-notifications being the master/coordinator that gets the tasks from the DB and a separate module being the worker/actor which is called by the master to process them. This pattern would be easier to test and expand for multiple types of work:
http://blog.gigaspaces.com/the-master-worker-pattern/
http://www.michaelpollmeier.com/akka-work-pulling-pattern